service/topology_coordinator.cc

```diff
@@ -1175,8 +1175,14 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
     }
 
     future<> generate_migration_updates(std::vector<canonical_mutation>& out, const group0_guard& guard, const migration_plan& plan) {
+        const auto& tables_waiting_for_resize_finalization = plan.resize_plan().finalize_resize;
         for (const tablet_migration_info& mig : plan.migrations()) {
             co_await coroutine::maybe_yield();
+            if (tables_waiting_for_resize_finalization.contains(mig.tablet.table)) {
+                // do not schedule migration for tables waiting for resize finalization
+                rtlogger.debug("Table {} waiting for resize finalization, ignoring migration: {}", mig.tablet.table, mig);
+                continue;
+            }
```

Comment:
@tgrabiec I tend to think we can give priority to resize execution over starting a new repair task, right? since the former executes quickly and contributes to system stability.

Reply:
Right, but this is already taken care of by the change to the scheduler not to emit new repair tasks.